virttest/utils_net.py

```diff
@@ -4790,12 +4797,14 @@ def create_linux_bridge_tmux(
                 "Create bridge fail as there is already interface named '%s' on the host "
                 "and can not delete with error: %s" % (linux_bridge_name, o)
             )
+
     if iface_name:
+        dhcp_cmd, _ = get_dhcp_client(session)
         shell_cmd = (
             "ip link add name {0} type bridge; ip link set {1} up; "
             "ip link set {1} master {0}; ip link set {0} up; "
-            "pkill dhclient; sleep 6; "
-            "dhclient {0};".format(linux_bridge_name, iface_name)
+            "pkill {2}; sleep 6; "
```

Comment:
I suggest use f-strings instead of .format()  for better readabilty.
E.g.
```py
f"pkill {dhcp_cmd}; sleep 6; "
```
The same for line 4851-4853